packages/template-retail-react-app/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 - Improved the layout of product tiles in product scroll and product list [#2446](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2446)
 - Updated 6 new languagues [#2495](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2495)
 - Show Bonus Product Label on OrderSummary component [#2524](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2524)
+- Implemented a voiceover feature in the email confirmation modal that cover all text including an alert for the close button and resend link button.[#2540](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/2540)
```

Comment:
I would update this to follow the same format as other a11y updates!

For example:
"[a11y] Hide svg from screenreader as they are decorative on homepage"